app/code/Magento/Backend/Controller/Adminhtml/Index/GlobalSearch.php

```diff
@@ -21,16 +26,33 @@ class GlobalSearch extends \Magento\Backend\Controller\Adminhtml\Index
     protected $_searchModules;
 
     /**
+     * modules that support preview
+     *
+     * @var array
+     */
+    private $previewModules;
+
+    /**
+     * @var ItemsFactory
+     */
+    private $itemsFactory;
+
+    /**
      * @param \Magento\Backend\App\Action\Context $context
      * @param \Magento\Framework\Controller\Result\JsonFactory $resultJsonFactory
      * @param array $searchModules
+     * @param array $previewModules
      */
     public function __construct(
         \Magento\Backend\App\Action\Context $context,
+        ItemsFactory $itemsFactory,
```

Comment:
New argument to constructor may only be added as the last on the list to preserve backwards compatibility.
Please refer to [Backwards Compatible Development](http://devdocs.magento.com/guides/v2.1/ext-best-practices/extension-coding/backwards-compatible-development/index.html#adding-a-constructor-parameter) guide for more information